### DISCUSSION TITLE
CLIO V EASYLINK

### DISCUSSION THREAD

**Discussion by h3nbx** [cedricp/ddt4all#1235] [Q&A]
Hello everyone,

I would like to activate features on my Clio 5 (easy link) such as wireless carplay, alarm activation, opening windows with the remote control, etc. 

Can anyone help me and explain the procedure? Is it true that we cannot connect directly with an OBD socket because there is a lock?

Thanks!

**Comment by AndyIreland** [comment]
hello - yes , go see discussion 
https://github.com/cedricp/ddt4all/discussions/956
on which discusses at length

**Comment by kartoffelpflanze** [comment]
As a summary, the Secure Gateway module only lets you change configurations through the normal OBD2 port if it is first unlocked by a scan tool with a special token and internet connection.

Many people instead opt to bypass the Gateway, by rerouting the ELM327's CAN wires to some place "after" the Gateway.

I find this tutorial is the easiest, even explains how to distinguish the CAN high and CAN low signals: [https://renault-zoe.forumpro.fr/t18076-guide-activer-apple-carplay-sans-fil-sur-les-zoe-52-r110-r135-bricoleurs-avertis-uniquement](https://renault-zoe.forumpro.fr/t18076-guide-activer-apple-carplay-sans-fil-sur-les-zoe-52-r110-r135-bricoleurs-avertis-uniquement)

I believe, but I'm not sure, that Wireless* Apple Carplay can be activated on any Easylink version. But I know for certain that Wireless* Android Auto can only be activated on the latest version, the one with the newest Renault logo, but it's very risky to update to it.

**Reply by AndyIreland** [reply]
> I believe, but I'm not sure, that Apple Carplay can be activated on any Easylink version. But I know for certain that Android Auto can only be activated on the latest version, the one with the newest Renault logo, but it's very risky to update to it.

Do you mean wireless Apple Carplay and Wireless Android Auto?  - as I thought as standard the easylink can do wired apple car play and wired android auto without modification?

**Reply by kartoffelpflanze** [reply]
Yes, of course. That's what the discussion is about.

**Comment by kucukkadir52** [comment]
Hello. I have a Clio 5. I wanted to unlock hidden features with the ecutweaker version of the ddt4all app. To do this, I first accessed the "SGW Gateway" by removing the glove compartment. The 16+32 cable helped me. I connected one end of the cable to the 32-pin port on the SGW and the other to the OBD device. I activated many features, such as the temperature and clock on the instrument panel and the window down function from the remote control, as shown in the attached image. However, I'm stuck at one point. I can read all the data in the "NAVIGATION" tab that controls EasyLink, but even when I press "Write All" and "Key On/Off Reset," the EasyLink system restarts, but no data changes. What can I do about this? Please help me?

**Reply by Renotech1** [reply]
If you did many hiden function. 
What is the function that dont work?

**Reply by kucukkadir52** [reply]
I can't unlock the hidden features in the EasyLink system.
